release-notes/sql-odbc.OpenSearch.release-notes-1.5.0.0.md

```diff
@@ -0,0 +1,43 @@
+## Version 1.5
+
+### Features
+
+* Add new parameter for SIGv4 auth. ([#42](https://github.com/opensearch-project/sql-odbc/pull/42))
```

Comment:
Can you call out the parameter name so users don't have to read the docs?

Reply:
Fixed in f9f9042.